nemo/collections/tts/data/dataset.py

```diff
@@ -453,6 +455,11 @@ def add_pitch(self, **kwargs):
             with open(Path(pitch_stats_path), 'r', encoding="utf-8") as pitch_f:
                 self.pitch_stats = json.load(pitch_f)
 
+    def add_speaker_embedding(self, **kwargs):
```

Comment:
Add a comment/method description as to what kind of speaker embedding will be imported, any specific format that is expected.